mmv1/products/cloudidentity/api.yaml

```diff
@@ -262,6 +262,16 @@ objects:
                 - :OWNER
                 - :MANAGER
                 - :MEMBER
+            - !ruby/object:Api::Type::NestedObject
+              name: 'expiryDetail'
+              description: |
+                The MembershipRole expiry details.
+              properties:
+                - !ruby/object:Api::Type::String
+                  name: 'expireTime'
```

Comment:
Would you mind marking this field as required? We tend to have at least one subfield of objects to be required to avoid users config an empty block.